Doc/library/idle.rst

```diff
@@ -713,6 +713,10 @@ or ``print`` or ``write`` to sys.stdout or sys.stderr,
 IDLE should be started in a command line window.  The secondary subprocess
 will then be attached to that window for input and output.
 
+The IDLE code running in the execution process adds frames to the call stack
+that would not be there otherwise.  IDLE wraps ``sys.getrecursionlimit`` and
+``sys.setrecursionlimit`` to reduce their visibility.
```

Comment:
"their" is ambiguous here; I suggest "to reduce the visibility of the additional stack frames."

Also, perhaps we should mention that this is not just about visibility? E.g. "to reduce the effect of the additional stack frames".